package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Mar 16 10:06:25 UTC 2018 - igonzalezsosa@suse.com
+
+- AutoYaST: fixed space distribution on LVM volume groups when
+  using percentages (bsc#1079369).
+
```

Comment:
Why to not increase version? More merges expected during the day?

Reply:
Because I have a second fix and I will bump the version later :)